src/dymaptic.GeoBlazor.Core/Scripts/arcGisJsInterop.ts

```diff
@@ -1075,4 +1110,32 @@ function waitForRender(viewId: string, dotNetRef: any): void {
             }
         }, 100);
     })
+}
+
+
+
+function buildDotNetListItem(item: ListItem): DotNetListItem | null {
+    if (item === undefined || item === null) return null;
+    let children: Array<DotNetListItem> = [];
+    item.children.forEach(c => {
+        let child = buildDotNetListItem(c);
+        if (child !== null) {
+            children.push(child);
+        }
+    });
+
+    /// <summary>
```

Comment:
I don't think this comment is needed here. Our library consumers will never call this code directly. It is only called by _our_ code above.